### ISSUE TITLE
Can't get cell value with right click

### ISSUE THREAD

**Issue by aliada77** [ragardner/tksheet#94]
When I select a cell with right click it doesn't give me the value of currently selected in the first place.  If I right click on same cell again then I get the value. However if I select different cell with right click, I get value of previous cell!

sheet.focus_set()
r, c = sheet.get_currently_selected()
cell_txt = sheet.get_cell_data(r, c)
print(cell_txt)

**Comment by ragardner** [comment]
I don't know how to reproduce this issue sorry, could you provide a bigger example?

It seems to work with the below code:

```python
from tksheet import Sheet
import tkinter as tk


class demo(tk.Tk):
    def __init__(self):
        tk.Tk.__init__(self)
        self.grid_columnconfigure(0, weight = 1)
        self.grid_rowconfigure(0, weight = 1)
        self.sheet = Sheet(self,
                            data = [[f"Row {r}, Column {c}" for c in range(10)] for r in range(10)]
                            )
        self.sheet.enable_bindings(("single_select",
                                         "drag_select",
                                    "select_all",
                                         "column_select",
                                         "row_select",
                                         "column_width_resize",
                                         "double_click_column_resize",
                                         "arrowkeys",
                                         "row_height_resize",
                                         "double_click_row_resize",
                                         "rc_select"
                                    ))
        self.sheet.grid(row = 0, column = 0, sticky = "nswe")
        self.sheet.bind("<3>", self.rc)

    def rc(self, event):
        print (event)
        r, c = self.sheet.get_currently_selected()
        cell_txt = self.sheet.get_cell_data(r, c)
        print (cell_txt)

        
app = demo()
app.mainloop()
```

**Comment by aliada77** [comment]
In your example right click menu or popup wasn't active. Try this code for better understanding my issue, specially when creating custom right click menu. Thanks!


```
from tksheet import Sheet
import tkinter as tk


class demo(tk.Tk):
    def __init__(self):
        tk.Tk.__init__(self)
        self.grid_columnconfigure(0, weight=1)
        self.grid_rowconfigure(0, weight=1)
        self.frame = tk.Frame(self)
        self.frame.grid_columnconfigure(0, weight=1)
        self.frame.grid_rowconfigure(0, weight=1)
        self.sheet = Sheet(self.frame,
                           data=[[f"Row {r}, Column {c}\nnewline1\nnewline2" for c in range(50)] for r in range(500)])
        self.sheet.enable_bindings(("single_select",
                                    "drag_select",
                                    "select_all",
                                    "column_select",
                                    "row_select",
                                    "column_width_resize",
                                    "double_click_column_resize",
                                    "arrowkeys",
                                    "row_height_resize",
                                    "double_click_row_resize",
                                    "right_click_popup_menu",
                                    "rc_select"
                                    ))
        self.sheet.popup_menu_add_command("Say Hello", self.new_right_click_button)
        self.frame.grid(row=0, column=0, sticky="nswe")
        self.sheet.grid(row=0, column=0, sticky="nswe")
        self.sheet.bind("<3>", self.rc)

    def rc(self, event):
        print(event)
        r, c = self.sheet.get_currently_selected()
        cell_txt = self.sheet.get_cell_data(r, c)
        print(cell_txt)

    def new_right_click_button(self, event=None):
        print("Hello World!")


app = demo()
app.mainloop()

```

**Comment by ragardner** [comment]
Thanks for reporting this and providing the examples,

It should be fixed now in `5.2.4`

**Comment by aliada77** [comment]
Great, thanks!
